cli/src/cli/commands/run.ts

```diff
@@ -166,6 +166,12 @@ export function runCommand(program: Command): Command {
       x => parseFloat(x),
       Options.defaultKgramsInWindow
     )
+    .option(
+      "-ic, --include-comments",
```

Comment:
```suggestion
      "-C, --include-comments",
```

The flag `-ic` feels like a combination of `-i`  and `-c` for me, and might be confusing.